Samples/Notifications/Push/cpp-console-unpackaged/cpp-console-unpackaged.cpp

```diff
@@ -0,0 +1,203 @@
+    {
+        auto channel{ result.Channel() };
+
+        DateTimeFormatter formater{ DateTimeFormatter(L"on {month.abbreviated} {day.integer(1)}, {year.full} at {hour.integer(1)}:{minute.integer(2)}:{second.integer(2)}") };
+
+        std::cout << "\nWNS ChannelURI: " << winrt::to_string(channel.Uri().ToString()) << std::endl;
+        std::wcout << L"\nThe WNS ChannelURI will expire " << formater.Format(channel.ExpirationTime()).c_str() << std::endl;
+
+        // It's the caller's responsibility to keep the channel alive
+        co_return channel;
+    }
+    else if (result.Status() == PushNotificationChannelStatus::CompletedFailure)
+    {
+        LOG_HR_MSG(result.ExtendedError(), "We hit a critical non-retryable error with the WNS ChannelURI request!");
+        co_return nullptr;
+    }
+    else
+    {
+        LOG_HR_MSG(result.ExtendedError(), "Some other failure occurred.");
+        co_return nullptr;
+    }
+
+};
+
+winrt::Microsoft::Windows::PushNotifications::PushNotificationChannel RequestChannel()
+{
+    auto task{ RequestChannelAsync() };
+    if (task.wait_for(std::chrono::seconds(300)) != AsyncStatus::Completed)
+    {
+        task.Cancel();
+        return nullptr;
+    }
+
+    auto result{ task.GetResults() };
+    return result;
+}
+
+// Subscribe to an event which will get signaled whenever a foreground notification arrives.
+void SubscribeForegroundEventHandler(const winrt::Microsoft::Windows::PushNotifications::PushNotificationChannel& channel)
+{
+    winrt::event_token token{ PushNotificationManager::Default().PushReceived([](auto const&, PushNotificationReceivedEventArgs const& args)
+        {
+        auto payload{ args.Payload() };
+
+            std::string payloadString(payload.begin(), payload.end());
+            std::cout << "\nPush notification content received from FOREGROUND: " << payloadString << std::endl;
+        }) };
+}
+
+int main()
+{
+    // An unpackaged app must initialize Dynamic Dependencies so we can consume the Windows App SDK APIs.
+    // See https://docs.microsoft.com/en-us/windows/apps/windows-app-sdk/tutorial-unpackaged-deployment?tabs=cpp,
+    // for details about using the WindowsAppSDK from an unpackaged app.
+    if (FAILED(LoadWindowsAppSDK()))
+    {
+        std::wcout << "\nCould not load Windows App SDK!" << std::endl;
+        std::cin.ignore();
+
+        return 1;
+    }
+
+    auto args{ AppInstance::GetCurrent().GetActivatedEventArgs() };
+    auto kind{ args.Kind() };
+    switch (kind)
+    {
+        // When it is launched normally (by the users, or from the debugger), the sample requests a WNS ChannelURI and
+        // displays it, then waits for notifications. This user can take a copy of the WNS ChannelURI and use it to send
+        // notifications to the sample
+        case ExtendedActivationKind::Launch:
+        {
+            // Request a WNS ChannelURI which can be passed off to an external app to send notifications to.
```

Comment:
Channel URI, not ChannelURI. Please address all occurrences.

Reply:
done

Reply:
Agreed.